Packs/Core/Playbooks/playbook-Unsigned_and_unpopular_process_performed_an_injection.yml

```diff
@@ -0,0 +1,852 @@
+id: Unsigned and unpopular process performed an injection
+version: -1
+name: Unsigned and unpopular process performed an injection
+description: "Overview\n\nThis playbook is designed to handle process injection alerts by analyzing preceding events, checking for agent prevention rules, and determining remediation actions. The playbook follows a logical flow to assess the risk, suggest improvements, and remediate potential threats.\n\n### This playbook handles the following alerts:\n \n- Unsigned and unpopular process performed injection into a commonly abused process\n\n- Unsigned and unpopular process performed process hollowing injection\n\n- Unsigned and unpopular process performed queue APC injection\n\n- Unsigned and unpopular process performed injection into a sensitive process\n\n- Unsigned and unpopular process performed injection into svchost.exe\n\n### Logical Flow\n\n#### 1. Retrieve Alerts from Case\n- Retrieve all alerts associated with the case.\n\n#### 2. Check for Agent Prevention Rule\n- Check if an agent prevention rule was triggered during the alert.\n  \n  ##### 2a. Block Mode\n  - If the agent rule was triggered and set to **block mode**, proceed directly to endpoint isolation.\n\n  ##### 2b. Report Mode\n  - If the agent rule was triggered and set to **report mode**, display a message suggesting the customer change the rule to **prevent mode** for future alerts.\n\n  ##### 2c. No Agent Rule Triggered\n  - If no agent prevention rule was triggered, continue to the following checks.\n\n#### 3. Check for Commonly Triggered Alerts Before Injection\n- Check if any commonly triggered alerts that typically precede process injection (as seen in previous cases) are present.\n\n  ##### 3a. Common Alerts Found\n  - If such alerts are found, initiate remediation actions.\n\n  ##### 3b. No Common Alerts Found\n  - If no such alerts are found, continue to the next step.\n\n#### 4. Analyze for MITRE Tactics TA0004 (Privilege Escalation) and TA0005 (Defense Evasion)\n- Check if there are any alerts or events that match the MITRE ATT&CK tactics for **Privilege Escalation (TA0004)** and **Defense Evasion (TA0005)**.\n\n  ##### 4a. Alerts Matching MITRE Tactics Found\n  - If alerts related to these tactics are found, initiate remediation actions.\n\n  ##### 4b. No MITRE Tactics Found\n  - If no relevant tactics are found, continue to the next step.\n\n#### 5. Check if Causality Process is Signed\n- Investigate if the causality (parent) process is signed with a legitimate certificate.\n\n  ##### 5a. Process is Signed\n  - If the process is signed by a trusted authority, close the alert.\n\n  ##### 5b. Process is Not Signed\n  - If the process is not signed, escalate for manual approval to proceed with remediation actions.\n\n## Remediation Actions\nWhen remediation is triggered:\n\n- **Terminate Malicious Processes**: Terminate the CGO.\n- **Endpoint Isolation**: Isolate the endpoint in specific high-risk cases to prevent further compromise."
```

Comment:
I think It's better to align with our other playbooks description, that format looks better only in the playbook editor tab, in every other place, it looks weird.

Reply:
Done